mpd/mpd.go

```diff
@@ -69,43 +70,42 @@ var (
 )
 
 type MPD struct {
-	XMLNs                      *string   `xml:"xmlns,attr"`
-	Scte35NS                   *Scte35NS `xml:"scte35,attr,omitempty"`
-	XsiNS                      *XsiNS    `xml:"xsi,attr,omitempty"`
-	XsiSchemaLocation          *XsiSL    `xml:"schemaLocation,attr,omitempty"`
-	Profiles                   *string   `xml:"profiles,attr"`
-	Type                       *string   `xml:"type,attr"`
-	MediaPresentationDuration  *string   `xml:"mediaPresentationDuration,attr"`
-	MinBufferTime              *string   `xml:"minBufferTime,attr"`
-	AvailabilityStartTime      *string   `xml:"availabilityStartTime,attr,omitempty"`
-	MinimumUpdatePeriod        *string   `xml:"minimumUpdatePeriod,attr"`
-	PublishTime                *string   `xml:"publishTime,attr"`
-	TimeShiftBufferDepth       *string   `xml:"timeShiftBufferDepth,attr"`
-	SuggestedPresentationDelay *Duration `xml:"suggestedPresentationDelay,attr,omitempty"`
-	BaseURL                    string    `xml:"BaseURL,omitempty"`
-	Location                   string    `xml:"Location,omitempty"`
+	XMLNs                      *string    `xml:"xmlns,attr"`
+	Scte35NS                   *Scte35NS  `xml:"scte35,attr,omitempty"`
+	XsiNS                      *XmlnsAttr `xml:"xsi,attr,omitempty"`
+	XsiSchemaLocation          *XsiSL     `xml:"schemaLocation,attr,omitempty"`
+	XsiCENC                    *XmlnsAttr `xml:"cenc,attr,omitempty"`
+	XsiMSPR                    *XmlnsAttr `xml:"mspr,attr,omitempty"`
```

Comment:
Quick q because idk enough about dash and drm, but this tag seems specific to PlayReady, was wondering if we were going to come across the same issues with Widevine?

Reply:
When looking at the P+ manifests, this covered the cases being outputted by Shaka for DASH + WV/PR

Reply:
@krystalmejia24  I believe that should be fine.. for widevine it's the cenc & pssh block contains the details abt key/algo etc... 
Q: @zsiec does this mean it will try to append this header on every manifest passing through bakery or does it have a way to check to only append these for drm manifests ?

Reply:
sounds good. Curious if the idea is to leave this in a branch and stack changes on top of it for bakery? Do we plan on getting these up to master branch?

Reply:
🤷🏽  Personally I don't like to leave branches like this. I would have preferred the other branch stay open until this is merged and we update the dependency sha on the bakery PR and a test case for this. 
But seems like that is merged already upstream... so upto Thomas I guess.

Reply:
Thanks.. perfect. I am just deploying bakery Image with this fix and can check a couple manifests and see how it works out.